hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml

```diff
@@ -144,6 +144,11 @@
       <artifactId>jersey-media-jaxb</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
How did you decide where to add jaxb-imp and with what scope ?

Reply:
I checked the usage of the `JettisonJaxbContext` in the projects where only used in the tests i added with test scope, otherwise i set no scope.